hack/generate-certs.example.yml

```diff
@@ -14,6 +14,8 @@ pprof-bind-addr: 0.0.0.0:2135
 
 # auth0 client id and domain is used to enable authentication flow
 # you can create a free dev account in Auth0 and create an app there
+# make sure to add your hosts to the allowed callback URLs list
+# for example, for "localhost" add "https://localhost"
```

Comment:
I think it's better to say that the address should include port there.

Reply:
Ah good point, added [a line about the port specification](https://github.com/siderolabs/omni/pull/601/commits/33ffb7a9ca2d4820c6e1f137dcb5f055802f2f11#diff-df763bddea6bdb595a01254092bf7317bcf857dbbaaa17b2b9b30b33e76e69baR19)